draft-eggert-bcp45bis.md

```diff
@@ -195,6 +202,8 @@ Chair is not otherwise involved in the operation of the SAA team - any SAA
 decision could be appealed to the IAB. The IAB shall then review the situation
 and attempt to resolve it in a manner of its own choosing.
 
+**RW: Just to check, is it right that appeals immediately goes to the IAB, and not the IESG first?**
+
```

Comment:
```suggestion
```
Based on the current BCP45, yes.

Reply:
Okay.  My thinking here was that behaviour on IETF mailing lists seems to currently fall under the IESG's responsibility rather than the IAB, and hence it seems slightly strange to appeal SAA decisions directly to the IAB.  But perhaps this is outside updating the current description of ietf@ and into the realm of how it should work ... and hence can be deferred?

Reply:
I would say so. If you wanted to change that, maybe see what gendispatch thinks?

Reply:
Okay.